crates/storage-query-datafusion/src/journal_events/schema.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2023 - 2025 Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use crate::table_macro::*;
+
+use datafusion::arrow::datatypes::DataType;
+
+define_sort_order!(sys_journal_events(partition_key, id));
+
+define_table!(sys_journal_events (
+    /// Internal column that is used for partitioning the services invocations. Can be ignored.
+    partition_key: DataType::UInt64,
```

Comment:
Is this actually needed or being used by anyone or any use case? It usually adds unnecessary clutter to the output. Maybe we can add it when need rises instead?

Reply:
I think this is required for the whole partition filter pushdown.